src/vcpkg/commands.build.cpp

```diff
@@ -1220,6 +1232,18 @@ namespace vcpkg
                 Hash::get_file_hash(fs, file, Hash::Algorithm::Sha256).value_or_exit(VCPKG_LINE_INFO));
         }
 
+        for (size_t i = 0; i < abi_info.pre_build_info->post_portfile_includes.size(); ++i)
+        {
+            auto& file = abi_info.pre_build_info->post_portfile_includes[i];
+            if (file.is_relative() || !fs.is_regular_file(file) || file.extension() != ".cmake")
```

Comment:
should the extensions check be dropped? CMake can include arbitrary files and if it is not in cmake syntax, cmake will complain about it.